tests/conftest.py

```diff
@@ -225,3 +226,29 @@ def variants_conda_build_sysroot(monkeypatch, request):
         ).stdout.strip(),
     )
     return request.param
+
+
+@pytest.fixture(scope="session")
+def conda_build_test_recipe_path(tmp_path_factory: pytest.TempPathFactory) -> Path:
+    """Clone conda_build_test_recipe.
+
+    This exposes the special dummy package "source code" used to test various git/svn/local recipe configurations.
+    """
+    # clone conda_build_test_recipe locally
+    repo = tmp_path_factory.mktemp("conda_build_test_recipe", numbered=False)
+    subprocess.run(
```

Comment:
could we fixture this subprocess command.. it will look cleaner

Reply:
not sure I understand, AFAIK no fixture offers subprocess functionality

Reply:
sorry re-phrase i meant in a util method instead.. since it is running a `git clone` command.. could we convert this to a util function and use it.. it is a possible common method for other tests..

Reply:
that is true..

Reply:
ok opting to leave this as is for now